assets/main.css

```diff
@@ -617,28 +617,31 @@ input:not([type=radio]),
   left: 8px;
 }
 
+.breadcrumbs-container {
+  margin: 80px auto;
+}
 .breadcrumbs-container .breadcrumbs-listings {
   display: flex;
   align-items: center;
-  gap: 12px;
-  margin: 40px auto;
+  gap: 8px;
+  margin-bottom: 24px;
 }
 .breadcrumbs-container .breadcrumbs-listings .first-listing {
-  color: #A8A8A8;
+  color: #9B9B9B;
   font-weight: 400;
-  font-size: 14px;
+  font-size: 16px;
   text-decoration: none;
 }
 .breadcrumbs-container .breadcrumbs-listings .second-listing {
   font-weight: 700;
   color: #000000;
-  font-size: 14px;
+  font-size: 16px;
   text-decoration: none;
 }
-[dir=ltr] .breadcrumbs-container .breadcrumbs-listings .listings__item img {
-  rotate: 180deg;
+.breadcrumbs-container .breadcrumbs-listings .listings__item {
+  color: #9B9B9B;
 }
 .breadcrumbs-container .breadcrumbs-name {
-  font-weight: 700;
-  font-size: 26px;
+  font-weight: 400;
```

Comment:
Shouldn't this be weight 700?

Reply:
In the design it does indeed say 400, however, since it's a different font the weight distribution is different as well, i.e: 400 in "Montserat" is 700 in "Almarai"

Reply:
in the design we have 400 so we already  have a setting in the theme editor for changing the font family.